CodeofConduct.md

```diff
@@ -45,34 +60,20 @@ Content must not be malware. For example, content (code, video, pictures, words,
 Content name, description, and other visible metadata must not include abusive, inappropriate, or harassing content.
 
 #Reporting Violations of this Code of Conduct
-If you believe someone is harassing you or has otherwise violated this Code of Conduct, please contact us at reach out to Mary Thengvall (@mary_grace), Aydrian Howard (@aydrian), or Rich Leland (@richleland) and send us an abuse report. If this is the initial report of a problem, please include as much detail as possible. It is easiest for us to address issues when we have more context.
+If you believe someone is harassing you or has otherwise violated this Code of Conduct, please reach out to Mary Thengvall (@mary_grace), Aydrian Howard (@aydrian), or Rich Leland (@richleland) and send us an abuse report. Please include as much detail as possible as it is easiest for us to address issues when we have more context.
 
-#Consequences
-All content published to SparkPost, including user account credentials, is hosted at the sole discretion of the SparkPost administrators.
+All complaints will be reviewed and investigated, and will result in a response that is deemed necessary and appropriate to the circumstances. The SparkPost administrators are obligated to maintain confidentiality with regard to the reporter of an incident. In general, we will choose the course of action that we judge as being most in the interest of fostering a safe and friendly community.
+
+You are also encouraged to contact us if you are curious about something that might be "on the line" between appropriate and inappropriate content. We are happy to provide guidance to help you be a successful part of our community.
 
-Unacceptable behavior from any community member, including sponsors, employees, customers, or others with decision-making authority, will not be tolerated.
+#Consequences
+Unacceptable behavior from any community member, including employees, customers, or others with decision-making authority, will not be tolerated.
 
 Anyone asked to stop unacceptable behavior is expected to comply immediately.
 
 If a community member engages in unacceptable behavior, the SparkPost administrators may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event or service).
 
-#Addressing Grievances
```

Comment:
there were so many sections saying essentially the same thing, so I
consolidated them into a clear & concise "here's what you do if you've been
offended" and "here are the consequences for offending" which I think will
work well.

and yes, oxford wards ;)

--
MARY THENGVALL
community manager
707.291.6220
@mary_grace
mary@sparkpost.com


*SparkPost*
developers.sparkpost.com
@SparkPostDev

On Mon, Aug 22, 2016 at 7:02 AM, Jason Rhodes <notifications@github.com>
wrote:

> In CodeofConduct.md
> <https://github.com/SparkPost/sparkpost.github.io/pull/136#discussion_r75682678>
> :
>
> >
> >  Anyone asked to stop unacceptable behavior is expected to comply immediately.
> >
> >  If a community member engages in unacceptable behavior, the SparkPost administrators may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning (and without refund in the case of a paid event or service).
> >
> > -#Addressing Grievances
>
> A+ for getting rid of this section -- anyone who wants to "address a
> grievance" will find their own way...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/SparkPost/sparkpost.github.io/pull/136/files/8d74ddb843996d1469d828425afedced109b4b1b#r75682678>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHqtqzhT5S-JmmQgB-5vumR1ZrjZt-Wxks5qiawNgaJpZM4Jo3AV>
> .
>